skin/base.css

```diff
@@ -301,6 +300,12 @@
   box-shadow: 0 0 0 1px Highlight inset;
 }
 
+#TabsToolbar #find-input.hide-search {
+  min-width: 0 !important;
+  border: none !important;
+  margin: 0 !important;
```

Comment:
I wonder if we should also disable it, so that people can't get in to a weird state when they hit `cmd-shift-l`…